framework/encode/d3d12_capture_manager.cpp

```diff
@@ -596,9 +596,12 @@ void D3D12CaptureManager::PrePresent(IDXGISwapChain_Wrapper* swapchain_wrapper)
     }
 }
 
-void D3D12CaptureManager::PostPresent(IDXGISwapChain_Wrapper* swapchain_wrapper)
+void D3D12CaptureManager::PostPresent(IDXGISwapChain_Wrapper* swapchain_wrapper, UINT flags)
 {
-    EndFrame();
+    if (flags != DXGI_PRESENT_TEST)
```

Comment:
Should this be `if ((flags & DXGI_PRESENT_TEST) == 0)` to handle the case where `flags` has other bits set with `DXGI_PRESENT_TEST`?

Reply:
correct